Gemfile

```diff
@@ -35,6 +35,8 @@ gem 'devise'
 
 gem 'pundit'
 
+gem 'activeadmin'
```

Comment:
Why did we add it 2 times?

Reply:
It was left while resolving conflicts, will remove it.